src/vuex/modules/delegation.js

```diff
@@ -126,145 +126,9 @@ export default ({ node }) => {
-
-      await dispatch(`sendTx`, {
-        type: `MsgDelegate`,
-        txArguments: {
-          toAddress: session.address,
-          delegator_address: session.address,
-          validator_address,
-          amount,
-          denom
-        },
-        gas,
-        gas_prices,
-        password,
-        submitType
-      })
-    },
-    async simulateUnbondingDelegation(
-      {
-        rootState: { stakingParameters, session },
-        dispatch
-      },
-      { validator, amount }
-    ) {
-      const denom = stakingParameters.parameters.bond_denom
-      return await dispatch(`simulateTx`, {
-        type: `MsgUndelegate`,
-        txArguments: {
-          toAddress: session.address,
-          delegator_address: session.address,
-          validator_address: validator.operator_address,
-          denom,
-          amount
-        }
-      })
-    },
-    async submitUnbondingDelegation(
-      {
-        rootState: { stakingParameters, session },
-        dispatch
-      },
-      { validator, amount, gas, gas_prices, password, submitType }
-    ) {
-      const denom = stakingParameters.parameters.bond_denom
-      await dispatch(`sendTx`, {
-        type: `MsgUndelegate`,
-        txArguments: {
-          toAddress: session.address,
-          delegator_address: session.address,
-          validator_address: validator.operator_address,
-          amount,
-          denom
-        },
-        gas,
-        gas_prices,
-        password,
-        submitType
-      })
-    },
-    async simulateRedelegation(
-      {
-        rootState: { stakingParameters, session },
-        dispatch
-      },
-      { validatorSrc, validatorDst, amount }
-    ) {
-      const denom = stakingParameters.parameters.bond_denom
-      return await dispatch(`simulateTx`, {
-        type: `MsgRedelegate`,
-        txArguments: {
-          toAddress: session.address,
-          delegator_address: session.address,
-          validator_src_address: validatorSrc.operator_address,
-          validator_dst_address: validatorDst.operator_address,
-          amount,
-          denom
-        }
-      })
-    },
-    async submitRedelegation(
-      {
-        rootState: { stakingParameters, session },
-        dispatch
-      },
-      {
-        validatorSrc,
-        validatorDst,
-        amount,
-        gas,
-        gas_prices,
-        password,
-        submitType
-      }
-    ) {
-      const denom = stakingParameters.parameters.bond_denom
-
-      await dispatch(`sendTx`, {
-        type: `MsgRedelegate`,
-        txArguments: {
-          toAddress: session.address,
-          delegator_address: session.address,
-          validator_src_address: validatorSrc.operator_address,
-          validator_dst_address: validatorDst.operator_address,
-          amount,
-          denom
-        },
-        gas,
-        gas_prices,
-        password,
-        submitType
-      })
-    }
+    async postSubmitDelegation() {},
```

Comment:
what are these for?

Reply:
Knowledge? Guess I'm breaking my own rule here aren't I? I'll remove.